tekton/assets/logs/tekton_tests.yaml

```diff
@@ -0,0 +1,68 @@
+id: "tekton"
+tests:
+ -
+  sample: |-
+    {
+      "severity" : "info",
+      "duration" : 0.002798625,
+      "caller" : "controller/controller.go:550",
+      "logger" : "tekton-pipelines-controller",
+      "commit" : "876b81e",
+      "message" : "Reconcile succeeded",
+      "timestamp" : "2024-02-28T08:15:18.164Z",
+      "knative" : {
+        "dev/kind" : "tekton.dev.TaskRun",
+        "dev/traceid" : "f9171cbc-d40e-4bb8-912a-8781b387cd97",
+        "dev/controller" : "github.com.tektoncd.pipeline.pkg.reconciler.taskrun.Reconciler",
+        "dev/key" : "tekton-pipelines/hello-task-run"
+      }
+    }
+  result:
+    custom:
+      caller: "controller/controller.go:550"
+      commit: "876b81e"
+      duration: 0.002798625
+      knative:
+        dev/controller: "github.com.tektoncd.pipeline.pkg.reconciler.taskrun.Reconciler"
+        dev/key: "tekton-pipelines/hello-task-run"
+        dev/kind: "tekton.dev.TaskRun"
+        dev/traceid: "f9171cbc-d40e-4bb8-912a-8781b387cd97"
```

Comment:
We have a standard property for trace_id. It would be great to remap on this one: https://docs.datadoghq.com/standard-attributes/?product=log+management&search=trace_id

Also, more information here: https://docs.datadoghq.com/logs/log_configuration/attributes_naming_convention/#reserved-attributes

Reply:
Thanks! That should be fixed

Reply:
https://docs.google.com/document/d/1RnMNhNJkKILft7RaHLwocZSyhBpMSE7YaQMI_cJQY_4/edit#heading=h.jcm9t1mt32oq
This is available in staging, feel free to test it #17118

Reply:
@FlorentClarret let me know when you test are done in staging and I can do a final check and approve the Pr!

Reply:
Looks good to me 🙂